copter/source/docs/traditional-helicopter-internal-rsc-governor.rst

```diff
@@ -4,29 +4,43 @@
 Internal RSC Governor
 =====================
 
-ArduPilot provides an internal RSC governor (:ref:`H_RSC_MODE<H_RSC_MODE>` =4) for those applications which do not have one externally to assure constant rotor head speed. This mode requires the setup of the ``H_RSC_THRCRV_x`` and ``H_RSC_GOV_x`` parameters and the use of an :ref:`RPM sensor<common-rpm>` for the rotor head speed.
+ArduPilot provides an internal RSC governor (:ref:`H_RSC_MODE<H_RSC_MODE>` =4) for those applications which do not have one externally to assure constant rotor head speed. This mode requires the setup of the ``H_RSC_THRCRV_x`` and ``H_RSC_GOV_x`` parameters and the use of an :ref:`RPM sensor<common-rpm>` for the rotor head speed.  The governor maintains desired rotor speed through a proportional controller adjusted through the droop compensator and a feedforward controller that uses the throttle curve to help respond to sudden loading and unloading of the rotor system. The governor is designed to handle changes in environmental conditions and even changes in the desired RPM from the nominal settings for which the throttle curve was originally tuned.  This is accomplished through the torque compensator which adjusts the reference from the throttle curve to maintain the desired rotor speed.  The governor is designed to maintain the desired RPM within the governor range.  If the RPM falls outside this range for more than 0.5 seconds, the governor will declare the appropriate overspeed or underspeed fault and the throttle output is reverted back to the throttle curve.  The motor interlock has to be disabled to reset the governor.  
```

Comment:
I dont think we talk anywhere about how to set the params actually used for throttle curve....is this set like an idleup curve , but since we have limited negative at low stick, kinda of lop-sided? probably should have a section/page and graph somewhere...I can work on that once I fully understand...

Reply:
The throttle curve is the same as what you would set in your transmitter for a given pitch curve. However unlike the transmitter which bases it from the position of the stick, ardupilot bases it from the limits of the collective blade pitch range given by H_COL_MIN to H_COL_MAX. So H_RSC_THRCRV_0 is the throttle output for collective at H_COL_MIN and H_RSC_THRCRV_100 is the throttle output for H_COL_MAX.

Reply:
so for aerobatic setups, you would set the curve like a V or U (as on a non governed vehicle) and in the stab modes since the collective is limited, it would still use the correct throttle curve point for low negative pitches....got it...